### PR TITLE
Print agent labels in debug mode (#4155)

### PR DESCRIPTION
**Pull request by 6543**
Backport  #4155

because right now there is no way to figure this out in any way

### CODE REVIEW

**Review by pat-s — COMMENTED** [review]
This is the same as #4155 ?

File: cmd/agent/core/agent.go [hunk 1]
```diff
@@ -238,6 +238,8 @@ func run(ctx context.Context, c *cli.Command, backends []types.Backend) error {
 		return err
 	}
 
+	log.Debug().Any("labels", labels).Msgf("agent configured with labels")
```
Comment (zc-devs) [inline cmd/agent/core/agent.go:241]: Maybe better to make it `info` and merge with line 285
> starting Woodpecker agent with version ... labels

Comment (6543) [reply cmd/agent/core/agent.go:241]: Normaly you as admin have set it so you should know this is for debugging porpuse

**Comment by 6543** [comment]
> This is the same as #4155 ?

Yes 1:1 backport

**Review by pat-s — APPROVED** [review]
Ah, missed the target branch...

**Pull request merged**